addons/repair/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2 import sql
+from openupgradelib import openupgrade
+
+
+_rename_columns = {
+    "repair_fee": [('invoice_line_id', None)],
+    "repair_line": [('invoice_line_id', None)],
+    "repair_order": [('invoice_id', None)],
+}
+
+
+def update_repair_fee_invoice_relation(env):
+    column_name = sql.Identifier(openupgrade.get_legacy_name("invoice_line_id"))
+    openupgrade.logged_query(
+        env.cr, sql.SQL(
+            """UPDATE repair_fee rf
+            SET {} = aml.id
```

Comment:
```suggestion
            SET invoice_line_id = aml.id
```

The problem you had before is because you had `SET rf.invoice_line_id` which is wrong. SET command never has the table, only the field.